trunner/tools/gdb.py

```diff
@@ -78,7 +78,8 @@ def run(self):
                 "gdb-multiarch",
                 encoding="ascii",
                 timeout=8,
-                cwd=self.cwd,
+                # pexpect required cwd to be provided as a string
```

Comment:
I know that it was my suggestion, but maybe it's not as much needed - too many comments decreases readability.
```suggestion
```